quodlibet/docs/development/plugins.rst

```diff
@@ -66,7 +66,7 @@ Creating a new Plugin
 
     class MyPlugin(EventPlugin):
         PLUGIN_ID = "myplugin"
-        PLUGIN_NAME = _("My Plugin")
+        PLUGIN_NAME = "My Plugin"
```

Comment:
The thing is, this isn't really the advice. There are currently just over 100 instances of this code `PLUGIN_NAME = (_"`, and _none_ of the uninternationalised one, so I'd suggest we leave this.

If you're having problems with this on _your_ plugin, you probably need to import `gettext` consistently with QL / conventions, see [the `quodlibet` base](https://github.com/quodlibet/quodlibet/blob/master/quodlibet/quodlibet/__init__.py#L21) for a hint.